CMakeLists.txt

```diff
@@ -106,23 +106,30 @@ include(cmake/SetupBasics.cmake)
 # Find third-party packages
 include(cmake/SetupPackages.cmake)
 
+# Set CUDA compute architecture
 if (RAJA_ENABLE_CUDA)
-  if (DEFINED CMAKE_CUDA_ARCHITECTURES)
-    if ("${CMAKE_CUDA_ARCHITECTURES}" STRLESS "35")
-      message( FATAL_ERROR "RAJA requires minimum CUDA compute architecture of 35")
-    endif()
-  else()
-    message(STATUS "CUDA compute architecture set to RAJA default 35 since it was not specified")
-    set(CMAKE_CUDA_ARCHITECTURES "35" CACHE STRING "Set CMAKE_CUDA_ARCHITECTURES to RAJA minimum supported" FORCE)
-  endif()
+  if (DEFINED CMAKE_CUDA_ARCHITECTURES AND NOT "${CMAKE_CUDA_ARCHITECTURES}" STREQUAL "")
```

Comment:
Using regex matching might be more robust against whitespace only strings. Something like `"${CMAKE_CUDA_ARCHITECTURES}" MATCHES "^\\s*$"`

Reply:
@adayton1 please post your comment on this PR https://github.com/LLNL/RAJA/pull/1892 as that is the one that will be merged. This one is on a fork and it would be best to make any changes on the branch in our repo.

Reply:
@victorapm I think I can make you a RAJA contributor. Then, you could make changes to branches in the RAJA repo. That would be best. I will let you know in a bit.